coalib/results/Result.py

```diff
@@ -41,12 +43,15 @@ def __init__(self,
                  debug_msg='',
                  diffs: (dict, None)=None,
                  confidence: int=100,
-                 aspect: (aspectbase, None)=None):
+                 aspect: (aspectbase, None)=None,
+                 message_arguments: dict={}):
         """
         :param origin:
             Class name or creator object of this object.
         :param message:
-            Message to show with this result.
+            Base message to show with this result.
+        :param message_arguments:
+            Arguments to be provided to the base message
```

Comment:
the docstring doesn't explain how to use them (i.e. that this is the python format syntax) - can we have a doctest with some nice code samples? That also reduces the complexity of the unittests